api/src/rsNcOpen.cpp

```diff
@@ -284,6 +284,7 @@ extern "C" {
 #endif // RODS_SERVER
                                 "api_nc_open",
                                 [](void*){},
+                                irods::clearOutStruct_noop, // Remove this line if targeting a 4.3.0 server.
```

Comment:
I think if we need to do another 4.3.0.x release, we can branch off before the changes in this PR.

Therefore, this comment can be removed.